### PR TITLE
Adding the metadata object so we can generate the mandate in the UK

### PR DESCRIPTION
**Pull request by danlong005**
Adding the metadata object allows us to have the mandate created in the UK.

### CODE REVIEW

File: lib/cashbox/model/metadata.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module Cashbox
+  class Metadata < Model
+    property "vin:MandateFlag", default: "1"
+    property "vin:MandateVersion", default: "1.0"
+    property "vin:MandateBankName", default: ""
+    property "vin:MandateID", default: (Time.now.to_f * 1000000).to_i.to_s
```
Comment (jonstorer) [inline lib/cashbox/model/metadata.rb]: what if -> `Digest::SHA2.hexdigest(subscription.id)[0..16]`

File: lib/cashbox/model/metadata.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module Cashbox
+  class Metadata < Model
+    property "vin:MandateFlag", default: "1"
+    property "vin:MandateVersion", default: "1.0"
+    property "vin:MandateBankName", default: "The Bank NA"
```
Comment (julamula) [inline lib/cashbox/model/metadata.rb]: this is a real bank name, though

Comment (danlong005) [reply lib/cashbox/model/metadata.rb]: fixed it

File: lib/cashbox/model/metadata.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module Cashbox
+  class Metadata < Model
+    property "vin:MandateFlag", default: "1"
+    property "vin:MandateVersion", default: "1.0"
+    property "vin:MandateBankName", default: "Bank name unavailable"
```
Comment (julamula) [inline lib/cashbox/model/metadata.rb]: 👍

**Pull request merged**